src/OrbitQt/main.cpp

```diff
@@ -161,6 +174,32 @@ void RunUiInstance(const DeploymentConfiguration& deployment_configuration,
   }
 }
 
+static QStringList ExtractCommandLineFlags(const std::vector<std::string>& command_line_args,
+                                           const std::vector<char*>& positional_args) {
+  QStringList command_line_flags;
+  absl::flat_hash_set<std::string> positional_arg_set(positional_args.begin(),
+                                                      positional_args.end());
+  for (std::string command_line_arg : command_line_args) {
+    if (!positional_arg_set.contains(command_line_arg)) {
+      command_line_flags << QString::fromStdString(command_line_arg);
+    }
+  }
+  return command_line_flags;
+}
+
+static std::vector<std::string> ExtractCaptureFilePaths(const std::vector<char*>& positional_args) {
```

Comment:
Iff this function stays (and is not removed): 
```suggestion
static std::vector<std::filesystem::path> ExtractCaptureFilePaths(const std::vector<char*>& positional_args) {
```

Reply:
Removed this function! Thanks for the suggestion!